Point-of-No-Return/Point-of-No-Return.vcxproj

```
@@ -78,12 +78,12 @@
     <LibraryPath>$(DXSDK_DIR)Lib\$(PlatformTarget);$(LibraryPath);Lib;Lib\$(PlatformTarget)\$(Configuration)</LibraryPath>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <IncludePath>$(DXSDK_DIR)Include;$(IncludePath);Include</IncludePath>
-    <LibraryPath>$(DXSDK_DIR)Lib\$(PlatformTarget);$(LibraryPath);Lib;Lib\$(PlatformTarget)\$(Configuration)</LibraryPath>
+    <IncludePath>$(DXSDK_DIR)Include;$(SolutionDir)point-of-No-Return\UI\ActionUI;$(SolutionDir)point-of-No-Return\Observer;$(SolutionDir)point-of-No-Return\Include;$(IncludePath)</IncludePath>

```

プロジェクトの設定なので、プロジェクトファイルからのpathを通せば良いのでは？
(下記はただのイメージですので、採用する場合は動作確認して下さいね)
```suggestion
    <IncludePath>$(DXSDK_DIR)Include;$(ProjectDir)point-of-No-Return\UI\ActionUI;$(ProjectDir)point-of-No-Return\Observer;$(Project)point-of-No-Return\Include;$(IncludePath)</IncludePath>
```